### PR TITLE
added xdai rpc endoint to remote config

### PR DESCRIPTION
**Pull request by poocart**
Changed xDai RPC endpoint and made it manageable through Remote Config.

### CODE REVIEW

File: src/constants/remoteConfigConstants.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const REMOTE_CONFIG = {
   PRISMIC_PRIVACY_POLICY_DOCUMENT_ID: 'prismic_privacy_policy_document_id',
   PRISMIC_TERMS_OF_POLICY_DOCUMENT_ID: 'prismic_terms_of_policy_document_id',
   PRISMIC_INTERJECTION_DOCUMENT_ID: 'prismic_interjection_document_id',
+  XDAI_RPC_ENDPOINT: 'xdai_rpc_endpoint',
```
Comment (IAmKio) [inline src/constants/remoteConfigConstants.js:49]:
```suggestion
  XDAI_RPC_ENDPOINT: 'app_xdai_rpc_endpoint',
```

For consistency 🙏

**Review by IAmKio — COMMENTED** [review]
Could we change that one value here please: https://github.com/pillarwallet/pillarwallet/pull/2786/files#diff-a4c7036e0e0573b69e522d1a0d8a0f89ce9ac5b48791cf21b985cee32431b71aR49

Then we're rocking and rolling!

File: src/services/opensea.js [hunk 2]
```diff
@@ -45,17 +45,21 @@ const getOpenSeaAssets = (
...
-      ? await getOpenSeaAssets(url, updatedPaginatedData, limit, newOffset)
-      : updatedPaginatedData;
+      const results = assets.length === limit
+        ? await getOpenSeaAssets(url, updatedPaginatedData, limit, newOffset)
+        : updatedPaginatedData;
 
-    resolve(results);
+      resolve(results);
+    } catch (err) {
+      reject(err);
```
Comment (IAmKio) [inline src/services/opensea.js:61]: Are we logging the error in the method that calls this?

Comment (poocart) [reply src/services/opensea.js:61]: Yes, call level up.

**Review by IAmKio — APPROVED** [review]

**Pull request merged**